tapa/graphir_conversion/utils.py

```diff
@@ -176,3 +180,49 @@ def get_task_graphir_parameters(task: Task) -> list[ModuleParameter]:
             )
         )
     return graphir_params
+
+
+def get_leaf_port_connection_mapping(
+    task_port: Port, task_module: Module, arg: str
+) -> dict[str, str]:
+    """Get leaf task port and slot port mapping.
+
+    Given a leaf task port and its arg, find all related ports in the task module based
+    on cat. Return a mapping from the leaf module port name to the connected slot port
+    name.
```

Comment:
This is basically creating a mapping that renames the RTL ports that corresponds to `task_port` as if they were generated by an argument named `arg` instead of `task_port .name`, right?